bucket/pterm.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "0.77",
+    "description": "a PuTTY-like wrapper program for Windows command prompts (or anything else running in a Windows console) that is not included in putty all-in-one archive/installer",
+    "homepage": "https://www.chiark.greenend.org.uk/~sgtatham/putty/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://the.earth.li/~sgtatham/putty/latest/w64/pterm.exe",
+            "hash": "sha512:10908538ccf762bf412ecf6ba8d8409ae723aead6f4911f9643c6f9b27bd19c6c85f287aa3fbb6cfe5ee9d20754a0a05781d6185eb09b487da09a3308e055eb5"
+        },
+        "32bit": {
+            "url": "https://the.earth.li/~sgtatham/putty/latest/w32/pterm.exe",
+            "hash": "sha512:4f7ed297dca1cb86da989671f3a3522dc8afa916845d6aea93641fdf1a677c2e0f0b4a5509fdf30ab290372cfaba413f5481107b0e715139129732eca5e4ad8b"
+        }
+    },
+    "bin": "pterm.exe",
+    "shortcuts": [
+    	[

```

```suggestion
        [
```
There is a TAB, which causes CI to fail.